WooCommerce/Classes/POS/Presentation/Reusable Views/POSHeaderLayoutConstants.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+enum POSHeaderLayoutConstants {
+    static let sectionTopPadding: CGFloat = 24
+    static let sectionBottomPadding: CGFloat = 24
```

Comment:
I kept these two separate for now, but we could collapse them on a `sectionVerticalPadding` as well.